CHANGELOG.md

```diff
@@ -1,15 +1,18 @@
-# Changelog
+## 2.0.2
+
+- Fixed a typo in a doc comment.
+- Added package topics to the pubspec file.
 
 ## 2.0.1
 
-Only zero out memory on successful allocation on Windows.
-Upgrade test dev dependency.
+- Only zero out memory on successful allocation on Windows.
+- Upgrade test dev dependency.
 
 ## 2.0.0
 
-Switch Windows memory allocation to use `CoTaskMemAlloc` and `CoTaskMemFree`,
-which will enable support for `NativeFinalizer`. This release requires Dart
-2.17.0 or greater.
+- Switch Windows memory allocation to use `CoTaskMemAlloc` and `CoTaskMemFree`,
+- which will enable support for `NativeFinalizer`. This release requires Dart
```

Comment:
This doesn't make sense, this is one long sentence, not bullet points.

Reply:
Ah, sorry, I missed that that this was one entry.

For future info, you can make edits to a PR - they show up in the UI as 'suggestions' - which can then be added to the PR as a new commit (it can help reduce latency in review cycles).